source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst

```diff
@@ -0,0 +1,147 @@
+
+   #. Click :guilabel:`Change`.
+
+   #. Confirm :guilabel:`Use as` is set to :guilabel:`Swap area`. See Figure 4.
+
+      .. figure:: figures/multi-boot-ubuntu-4.png
+
+         Figure 4: Ubuntu - Set swap partition
+
+#. Follow the remaining prompts to complete the installation of Ubuntu.
+
+#. At this point, the ability to boot |CL| is lost because `Grub`
+   is the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add Ubuntu as a boot option.
+
+   #. Boot into Ubuntu.
+
+   #. Log in.
+
+   #. Get root permissions.
+
+      .. code-block:: console
+
+        $ sudo -s
+
+   #. Locate the Ubuntu :file:`grub.cfg` file in the :file:`/boot/grub/`
+      directory and look for the :guilabel:`menuentry` section. The
+      highlighted lines identify the kernel, the :file:`initrd` files, the
+      root partition UUID, and the additional parameters used. Use this
+      information to create a new Systemd-Boot entry for Ubuntu. See Figure 5.
+
+      .. figure:: figures/multi-boot-ubuntu-5.png
+
+         Figure 5: Ubuntu: grub.cfg
+
+   #. Copy the kernel and :file:`initrd` to the EFI partition.
+
+      .. code-block:: console
+
+         # cp /boot/vmlinuz-4.8.0-36-generic.efi.signed /boot/efi
+
+         # cp /boot/initrd.img-4.8.0-36-generic /boot/efi
+
+   #. Create a boot entry for Ubuntu. The file must contain at least these
+      settings:
+
+      +---------+------------------------------------+
+      | Setting | Description                        |
+      +=========+====================================+
+      | title   | Text to show in the boot menu      |
+      +---------+------------------------------------+
+      | linux   | Linux kernel image                 |
+      +---------+------------------------------------+
+      | initrd  | initramfs image                    |
+      +---------+------------------------------------+
+      | options | Options to pass to the EFI program |
+      |         | or kernel boot parameters          |
+      +---------+------------------------------------+
+
+      See the `systemd boot loader documentation`_ for additional
+      details.
+
+      The *options* parameters must specify the root partition UUID and
+      any additional parameters that Ubuntu requires.
+
+      .. note:: The root partition UUID used below is unique to this example.
+
+      .. code-block:: console
+
+         # cd /boot/efi/loader/entries
+
+         # vi ubuntu.conf
+
+      Add the following lines to the :file:`ubuntu.conf` file:
+
+      .. code-block:: console
+
+         title Ubuntu 16.04 LTS Desktop
+
+         linux /vmlinuz-4.8.0-36-generic.efi.signed
+
+         initrd /initrd.img-4.8.0-36-generic
+
+         options root=UUID=17f0aa66-3467-4f99-b92c-8b2cea1045aa ro
+
+#. Re-install Systemd-Boot to make it the default boot loader.
+
+   .. code-block:: console
+
+      # bootctl install --path /boot/efi
```

Comment:
Change "# bootctl install --path /boot/efi" to "$ sudo bootctl install --path /boot/efi"

Reply:
Text edits deferred to next phase of updates.